c7n/resources/rdscluster.py

```diff
@@ -216,5 +211,141 @@ def process(self, snapshots):
     def process_snapshot_set(self, snapshots_set):
         c = local_session(self.manager.session_factory).client('rds')
         for s in snapshots_set:
-            c.delete_db_cluster_snapshot(
-                DBClusterSnapshotIdentifier=s['DBClusterSnapshotIdentifier'])
+            try:
+                c.delete_db_cluster_snapshot(
+                    DBClusterSnapshotIdentifier=s['DBClusterSnapshotIdentifier'])
+            except ClientError as e:
+                raise
+
+
+@RDSClusterSnapshot.action_registry.register('restore')
+class RDSClusterSnapshotRestore(BaseAction):
+    schema = type_schema(
+        'restore', vpc_sec_groups={'type': 'array'}, port={'type': 'number'}, db_subnet_group_name={'type': 'string'},
+        snapshot_identifier={'type': 'string'})
+
+    def process(self, resources):
+
+        vpc_sec_groups = self.data.get('vpc-security-groups')
+        port = self.data.get('port')
+        db_subnet_group_name = self.data.get('db-security-group-name')
+        snapshot_id = self.data.get('snapshot-identifier');
+
+        for latest_snapshot in self.get_latest_snapshot(resources):
+            client = local_session(self.manager.session_factory).client('rds')
+
+            if vpc_sec_groups is None:
+                self.log_default_value('vpc_sec_group')
+            elif db_subnet_group_name is None:
+                self.log_default_value('db_subnet_group_name')
+            elif port is None:
+                self.log_default_value('port')
+            else:
+                log.info('USING PARAMS PROVIDED IN POLICY')
+
+            print 'USING SNAPSHOT FOR CLUSTER::', latest_snapshot['DBClusterIdentifier']
```

Comment:
yeah, sorry i was debugging something and log.debug or log.info wasnt printing in the format i was looking for.  Just forgot to remove this print statement